core/source/Test/Html.mint

```diff
@@ -10,9 +10,8 @@ module Test.Html {
     (() => {
       let root = document.createElement('div')
       document.body.appendChild(root)
-      ReactDOM.render(#{node}, root)
+      _render(#{node}, root)
       return new TestContext(root, () => {
-        ReactDOM.unmountComponentAtNode(root)
```

Comment:
Not too sure about this. I mentioned something similar in https://github.com/mint-lang/mint-runtime/pull/8#issuecomment-588424638.

Reply:
We need to keep this functionality somehow so tests are run in a clean state: components needs to unsubscribe from providers and stores (which runs in `componentWillUnmount`) does Preact has something which can do this?

Reply:
I think I found out how they do it: https://github.com/preactjs/preact/blob/master/compat/src/index.js#L67